corehq/apps/users/models.py

```diff
@@ -2374,12 +2377,40 @@ def is_member_of(self, domain_qs):
 #
 # Django  models go here
 #
-class Invitation(Document):
+class SQLInvitation(SyncSQLToCouchMixin, models.Model):
+    email = models.CharField(max_length=100, db_index=True)
```

Comment:
I don't see any DB migrations for these new models. Also FYI its better to add the index as a separate step since diong it here will create two indexes, one for exact matches and one for partial matches (which we obviously don't need). This is dump but that's the way South does it.

Also maybe sync with @emord about whether you should use Django migrations instead of South.